taotie/arxiv_author.json

```diff
@@ -0,0 +1,7 @@
+[
+  "affiliation-1": ["author1", "author2"],
+  "affiliation-2": ["author3", "author4", "author5"],
+  "affiliation-3": ["author6", "author7", "author8"],
+  ...
+]
+
```

Comment:
Please use real data that are currently available in .env. You can put all of the existing authors into one dummy affiliation.